dashboard/test/lib/api/v1/school_autocomplete_test.rb

```diff
@@ -20,4 +22,54 @@ class Api::V1::SchoolAutocompleteTest < ActiveSupport::TestCase
   test 'is zip search with invalid extended partial zip' do
     assert_equal false, Api::V1::SchoolAutocomplete.search_by_zip?('12345-ab')
   end
+
+  test 'search by exact unique name matches only 1 school' do
+    search_results = Api::V1::SchoolAutocomplete.get_matches('SEQUOYAH SCH - CHALKVILLE CAMPUS', MAXIMUM_RESULTS, false)
```

Comment:
Do I need to seed my db in a certain way to be able to run these tests?

Reply:
These tests look good, thanks for adding them! They depends on an exact set of data on test database so we should call out what setup we depend on here, or better yet, create our own data in this tests so there is no external dependency.

Reply:
These tests rely on [Fixtures](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/fixtures/schools.tsv), so they're accessible via standard Rails conventions (for example, set `RAILS_ENV=test` when starting the dashboard console).